pom.xml

```diff
@@ -3,7 +3,7 @@
 	<modelVersion>4.0.0</modelVersion>
 	<groupId>com.redislabs</groupId>
 	<artifactId>spark-redis_2.11</artifactId>
-	<version>2.5.0-SNAPSHOT</version>
+	<version>2.6.0-SNAPSHOT</version>
```

Comment:
Why do you update the version to 2.6.0 on 2.4 branch?

Reply:
branch-2.4 is for Spark 2.4, there are spark-redis 2.4.x and 2.5.x versions there. 
spark-redis 2.6.0 should be also there as it's built against Spark 2.4